app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,87 @@
+class TweetsController < ApplicationController
+
+
+	get '/tweets' do
+		if logged_in?
+			@user = current_user
+			success
+			erb :'/tweets/tweets'
+		else
+			please_login
+			redirect '/login'
+		end
+	end
+
+
+	get '/tweets/new' do
+		if logged_in?
+			@user = current_user
+			erb :'/tweets/create_tweet'
+		else
+			please_login
+			redirect '/login'
+		end
+	end
+
+	post '/tweets/new' do
+		if filled_out(params)
+			@tweet = Tweet.create(content: params[:content])
+			@tweet.id = Tweet.all.last.id
+			@tweet.user_id = current_user.id
+			@tweet.save
+			success
+			redirect '/tweets'
+		else
+			not_filled_out
+			redirect '/tweets/new'
+		end
+	end
+
+
+	get '/tweets/:id' do
+		if logged_in?
+			@tweet = Tweet.find(params[:id])
+			erb :'/tweets/show_tweet'
+		else
+			please_login
+			redirect '/login'
+		end
+	end
+
+
+	get '/tweets/:id/edit' do
+		if logged_in?
+			@tweet = Tweet.find(params[:id])
+			erb :'/tweets/edit_tweet'
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?